_posts/2020-05-15-state-of-presto.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  "State of Presto"
+author: Manfred Moser
+excerpt_separator: <!--more-->
+---
+
+Presto is continuing to gain adoption across many industries and use cases. Our
+community is growing rapidly and generally there is a lot going on. So we are
```

Comment:
maybe remove `generally`

Reply:
Starting a sentence with "So" seems bad, and the last sentence is a fragment, so let's combine

> Our community is growing rapidly and there is a lot going on, so we are taking the Presto Summit online, starting with a State of Presto webinar with the founders of the project.

Reply:
I used that but broke it up into two sentences since it got too long..